test/pytest/test_pytorch_profiler.py

```diff
@@ -15,15 +15,15 @@
 import test_utils
 from concurrent import futures
 
-REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
-data_file_mnist = os.path.join(REPO_ROOT, "examples/image_classifier/mnist/test_data/1.png")
+REPO_ROOT = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../"))
```

Comment:
We can improve this line too

Reply:
```suggestion
REPO_ROOT = os.path.normpath(
    os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
)
```

Again, I can afford only that to stay readable and pythonic as well